Box bbox
float32 confidence
int32 class_id